doc/users/whats_new/style_changes.rst

```diff
@@ -28,6 +28,13 @@ Colors
 - Grid lines are light grey solid 1pt lines.  They are no longer dashed by
   default.
 
+Plots
+`````
+
+- The default size of the elements in a scatter plot is now based on
+  the rcParam ``lines.markersize`` so it is consistent with ``plot(X,
+  Y, 'o')``.  The old value was 20.
```

Comment:
The new value is 36 (6 ^ 2), but it's not hardcoded -- it comes from `lines.markersize`.